submissions/Roophee/JS - DOM/index.js

```diff
@@ -0,0 +1,138 @@
+    },
+    "pantheon": {
+        "title": "The Pantheon: The ancient building still being used after 2,000 years",
+        "imgurl": "img/panth.jpg",
+        "alt": "pantheon",
+        "text": " When visitors walk into the Pantheon in Rome and encounter its colossal dome, they may experience the same theatricality as its guests nearly 2,000 years ago.\"Anyone who steps inside the Pantheon immediately feels the crushing weight of human history, but also the incredible lightness of human creativity,\" said John Ochsendorf, professor of architecture at MIT and former director of the American Academy in Rome. \"You come into this grand space and you look up and you see the sky or a passing cloud. And you think: \'How could they have done this nearly two millennia ago?\'\" The Pantheon is the oldest building in the world that's still in use today. Since the 7th century, it has been a Roman Catholic church. Built around 125 A.D. by the Roman emperor Publius Aelius Hadrianus, it was actually the third iteration of the structure. The first Pantheon caught fire around 80 A.D. and was rebuilt shortly after, but it was struck by lightning and burned down again around 110 A.D. The buildings' ill fate led to rumors that the Pantheon was cursed. The facade of the completed structure riffed on ancient Greek motifs, with a portico entrance featuring a pediment -- a triangular top -- and two rows of Corinthian columns. The interior was sweeping and airy, capped by a dome that to this day -- is still the largest unsupported concrete dome in the world."
+    },
+};
+
+
+// CREATE NAV
+
+let dynamicNav = function (e) {
+    e.preventDefault();
+    let fragment = document.createDocumentFragment();
+
+    for (let i of Object.keys(contentStorage)) {
+        let navItem = document.createElement("li");
+        let navLink = document.createElement("a");
```

Comment:
You can use template string for inner part of navItem element instead of creating additional elements or unnecessary operation such as append.

Reply:
Done.